src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -772,24 +776,24 @@ public void onFailure(Throwable caught) {
 
 	private void configureTableCommands() {
```

Comment:
Most of this PR is modifying where we have branching logic regarding Datasets and deciding to do the same thing or different things for Dataset Collections